parm/data_locations.yml

```diff
@@ -78,6 +78,7 @@ FV3GFS:
       - /NCEPPROD/hpssprod/runhistory/rh{yyyy}/{yyyymm}/{yyyymmdd}
       - /NCEPPROD/hpssprod/runhistory/rh{yyyy}/{yyyymm}/{yyyymmdd}
       - /NCEPPROD/hpssprod/runhistory/rh{yyyy}/{yyyymm}/{yyyymmdd}
+      - /NCEPPROD/hpssprod/runhistory/rh{yyyy}/{yyyymm}/{yyyymmdd}
```

Comment:
Just curious, why is this line repeated?

Reply:
@JeffBeck-NOAA, in the current structure of `ush/retrieve_data.py`, each 'archive_file_names` line requires the corresponding line in the `archive_path`. @christinaholtNOAA or @danielabdi-noaa may have an answer to this.